lib/analysis_options.yaml

```diff
@@ -77,6 +78,7 @@ linter:
     avoid_return_types_on_setters: true # flutter
     avoid_returning_null_for_void: true # flutter
     avoid_returning_this: true
+    avoid_setters_without_getters: false
```

Comment:
I don't see a reason why we should disable this. If you want to change values through custom setter overrides why not use a function instead that applies a modification.

Reply:
use community default

Reply:
so this is the community default